Gemfile

```diff
@@ -130,6 +136,9 @@ group :development, :test do
   gem 'bullet', '>= 4.14.9'
 
   gem 'parallel_tests'
+
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platform: :mri
```

Comment:
:+1: I have been adding this for the longest time >_< Should have done this a long time ago.